utils/update_checkout/update-checkout-config.json

```diff
@@ -469,30 +469,30 @@
         "tensorflow": {
             "aliases": ["tensorflow"],
             "repos": {
-                "llvm": "swift-DEVELOPMENT-SNAPSHOT-2019-10-08-a",
+                "llvm": "swift-DEVELOPMENT-SNAPSHOT-2019-10-13-a",
```

Comment:
I noticed that commit hashes for `clang` for `lldb` have not updated - I wonder if that is intentional (perhaps there was nothing new to merge)?

Reply:
Thanks for the catch @dan-zheng ! I tried to update them, but those repos are frozen due to the monorepo transition. So I'm going to leave them frozen for now. Does that sound okay?

Reply:
If the toolchain builds and passes tests without updating those repos, then we got lucky and that seems fine to me.

If we realize that we do need updates in those repos, options are:
* Merge past the monorepo transition, transition ourselves to the monorepo, and make the updates in the monorepo.
* Temporarily upload the updates to personal forks of the repos and point update-checkout-config there. After we transition to the monorepo, this temporary hack disappears.